### PR TITLE
Introducing utils.helpers for miscellaneous helper functions

### PR DESCRIPTION
**Pull request by tilmankamp**

### CODE REVIEW

File: util/helpers.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+
+def keep_only_digits(txt):
+    return ''.join(filter(lambda c: c.isdigit(), txt))
```
Comment (reuben) [inline util/helpers.py]: nit: `str.isdigit` instead of `lambda c: c.isdigit()`.

Comment (tilmankamp) [reply util/helpers.py]: Thought the same - didn't work as `self` is a parameter that also has to be bound.

Comment (reuben) [reply util/helpers.py]: > In [1]: ''.join(filter(str.isdigit, 'aeiou1234'))
> Out[1]: '1234'

Comment (reuben) [reply util/helpers.py]: `self` will be bound when called by filter, it's just the first parameter.

In [4]: str.isdigit('1')
Out[4]: True

In [5]: str.isdigit('a')
Out[5]: False

Comment (tilmankamp) [reply util/helpers.py]: Ah - got it.

**Pull request merged**